lms/static/sass/elements/_system-feedback.scss

```diff
@@ -722,7 +722,7 @@
 }
 
 // adopted alerts
-.alert {
+.alert:not(.pattern-library-shim) {
```

Comment:
I had to add this to prevent my new element from getting these styles - previously the additional specificity of the wrapping `.pattern-library-shim` took care of this.